### PR TITLE
update ruby versions

### PR DESCRIPTION
**Pull request by bkmgit**
addressing https://github.com/phusion/passenger-docker/issues/315
following https://github.com/phusion/passenger-docker/commit/323262874d47f7f243b387c61b6ccc53257bcb5f

### CODE REVIEW

**Comment by bkmgit** [comment]
# Build
```
$ make build_ruby26
$ docker run -it phusion/passenger-ruby26:1.0.16 ruby -v
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
ruby 2.6.8p205 (2021-07-07 revision 67951) [x86_64-linux]
```

```
$ make build_ruby27
$ docker run -it phusion/passenger-ruby27:1.0.16 ruby -v
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
ruby 2.7.4p191 (2021-07-07 revision a21a3b7d23) [x86_64-linux]
```

```
$ make build_ruby30
$ docker run -it phusion/passenger-ruby30:1.0.16 ruby -v
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux]

```

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Basics (learn more at [baseimage-docker](http://phusion.github.io/baseimage-dock
 
 Language support:
 
- * Ruby 2.4.10, 2.5.9, 2.6.7, 2.7.3, 3.0.1 and JRuby 9.2.13.0.
+ * Ruby 2.4.10, 2.5.9, 2.6.8, 2.7.4, 3.0.2 and JRuby 9.2.13.0.
    * RVM is used to manage Ruby versions. [Why RVM?](#why_rvm)
    * 2.7.3 is configured as the default.
```
Comment (pboling) [inline README.md:99]: Should use 2.7.4 as the default

**Review by pboling — CHANGES_REQUESTED** [review]
Needs (at least) one additional small change from 2.7.3 to 2.7.4

File: README.md (hunk 1 above)
Comment (bkmgit) [reply README.md:99]: Thanks

**Pull request merged**